### DISCUSSION TITLE
Transparent background setting failure

### DISCUSSION THREAD

**Discussion by TaterDoge** [ayamir/nvimdots#450] [Q&A]
path: neovim/lua/modules/ui/config.lua         catppuccin this function
code: local transparent_background = true -- Set background transparency here!

**Comment by Jint-lzxy** [comment]
Can you [open an issue](https://github.com/ayamir/nvimdots/issues/new?assignees=&labels=bug&template=bug_report.yml) to describe your problem in detail? This discussion doesn't provide any useful information.
